adapter/src/model/FormatConfig.ts

```diff
@@ -0,0 +1,22 @@
+import { Format } from './enum/Format';
+
+export class FormatConfig {
```

Comment:
Could be pulled up to `AdapterConfig` => removes Java file clutter

Reply:
What do you mean with could be pulled up? we are not sure

Reply:
I mean moving it to the `AdapterConfig` file since it is only a very small interface definition.